alpha/declcfg/load.go

```diff
@@ -168,3 +168,20 @@ func readYAMLOrJSON(r io.Reader) (*DeclarativeConfig, error) {
 	}
 	return cfg, nil
 }
+
+func LoadFile(root fs.FS, file fs.File, path string) (*DeclarativeConfig, error) {
+	var (
+		cfg = &DeclarativeConfig{}
+		err error
+	)
+
+	if cfg, err = readYAMLOrJSON(file); err != nil {
+		return nil, err
+	}
+
+	if err = readBundleObjects(cfg.Bundles, root, path); err != nil {
```

Comment:
```suggestion
	if err := readBundleObjects(cfg.Bundles, root, path); err != nil {
```

Reply:
Yes.  Committed after testing both new `veneer render` and "regular" render opm commands.

Reply:
Seems like this suggestion and fix may have been lost?